<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="sliceLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Slice</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pdLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Persistence Diagram</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Data File</string>
    </property>
   </widget>
   <widget class="QLabel" name="homDimLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Homology Dimension:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="homDimSpinBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="computeButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="offsetLabel">
       <property name="text">
        <string>&amp;Offset:</string>
       </property>
       <property name="buddy">
        <cstring>offsetSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="angleLabel">
       <property name="text">
        <string>&amp;Angle:</string>
       </property>
       <property name="buddy">
        <cstring>angleSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="angleSlider">
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="offsetSpinBox"/>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="angleSpinBox">
       <property name="maximum">
        <number>90</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="offsetSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="SliceArea" name="sliceArea" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="PDArea" name="pdArea" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="normCoordCheckBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>470</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Normalize Coordinates</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>530</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cleanDiagLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Clean Diagonal</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaleLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>530</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current/Default Scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fitScalePushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>560</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit Scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetScalePushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>560</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_Window">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMain_Window"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SliceArea</class>
   <extends>QWidget</extends>
   <header>slicearea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PDArea</class>
   <extends>QWidget</extends>
   <header>pdarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
